### PR TITLE
[stable25] Fix/35886 replace custom tooltips with native one for personal and administration settings stable25

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
- [x] :warning: This backport had conflicts and is incomplete

backport of #35901

## ToDo

- [x] : resolve conflicts in BackgroundSettings.vue

### CODE REVIEW

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: apps/settings/src/components/PersonalInfo/shared/FederationControlAction.vue [hunk 1]
```diff
@@ -27,9 +27,8 @@
 		:close-after-click="true"
 		:disabled="!isSupportedScope"
 		:icon="iconClass"
-		:title="displayName"
+		:title="isSupportedScope ? tooltip : tooltipDisabled"
 		@click.stop.prevent="updateScope">
-		{{ isSupportedScope ? tooltip : tooltipDisabled }}
```
Comment (nickvergessen) [inline apps/settings/src/components/PersonalInfo/shared/FederationControlAction.vue:32]: Now the button has no text and no icon anymore?!

File: apps/theming/src/components/BackgroundSettings.vue [hunk 2]
```diff
@@ -67,10 +67,16 @@
 		<!-- Background set selection -->
 		<button v-for="shippedBackground in shippedBackgrounds"
 			:key="shippedBackground.name"
+<<<<<<< HEAD
```
Comment (nickvergessen) [inline apps/theming/src/components/BackgroundSettings.vue:70]: 👀

File: apps/settings/src/components/PersonalInfo/shared/FederationControlAction.vue (hunk 1 above)
Comment (nickvergessen) [reply apps/settings/src/components/PersonalInfo/shared/FederationControlAction.vue:32]: Ah, this is the thing I tried to find recently which breaks the profile fields in master:
Before | After
---|---
Bildschirmfoto vom 2023-01-17 22-08-38 | Bildschirmfoto vom 2023-01-17 22-09-07

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Comment by JuliaKirschenheuter** [comment]
@nickvergessen 

Thank you! I will fix that problem as a follow-up ticket on master + will create a follow-up backport to stable25. Could we merge this PR or should i just close this one?

**Comment by nickvergessen** [comment]
You can also just restore the deleted line in this pr and then we merge?
The we don't have to backport a second pr?

File: apps/federatedfilesharing/src/components/PersonalSettings.vue [hunk 3]
```diff
@@ -24,7 +24,8 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 			{{ t('federatedfilesharing', 'Your Federated Cloud ID:') }}
 			<strong id="cloudid">{{ cloudId }}</strong>
 			<NcButton ref="clipboard"
-				v-tooltip="copyLinkTooltip"
+				:title="copyLinkTooltip"
+				:aria-label="copyLinkTooltip"
 				class="clipboard"
 				type="tertiary-no-background"
 				@click.prevent="copyCloudId">
```
Comment (nickvergessen) [inline apps/federatedfilesharing/src/components/PersonalSettings.vue:31]: Something for a follow up, but there is no feedback on this. Others have a toast notification top right to say `Copied!` etc.

**Review by nickvergessen — APPROVED** [review]

File: apps/federatedfilesharing/src/components/PersonalSettings.vue (hunk 3 above)
Comment (JuliaKirschenheuter) [reply apps/federatedfilesharing/src/components/PersonalSettings.vue:31]: Thanks: https://github.com/nextcloud/server/pull/36250

**Comment by ChristophWurst** [comment]
> fixup! build locally

FYI this is not how fixup commits are structured. Git won't be able to resolve what commit you are fixing up.

**Review by ChristophWurst — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Pull request merged**